### ISSUE TITLE
CSV files corrupt risk of loosing age on day.

### ISSUE THREAD

**Issue by drnickw** [rogernclarke/Rankings#18]
Checkout race 16675, several riders dont have a category or club on the HTML results although they have category on the PDF but still no club. these riders do have a CTT number but the CSV file completely ignores them! (ie no record at all)
I had been using CSV as our prime data source and only getting the HTML or PDF if coumn data was missing but this turns it on its head. Looks like HTML may be the safest source of primary data but then its going to be hard to get the age on day, especially if random records are dropped!

**Comment by rogernclarke** [comment]
I wonder if this relates to them not being shown on the start sheet on the CTT site.
There is some strange tie-up behind the scenes on the CTT site. I reported an error in their rankings this morning and I have just had a reply which "explains" that it is to do with paper entries. They have fixed the error I spotted, sort of. Separate to that issue, my rankings total is correct but the counting events shown aren't the top 12 scores!
I think you are going to have to drive the results from the most reliable source for a complete set of rides, then add data to those records if missing from the primary source.
